go linter issues

Linter warns what some variables should have different names:

```
velobike/authorize.go:11:2: struct field SessionId should be SessionID
velobike/history.go:16:2: struct field Id should be ID
velobike/parkings.go:17:2: struct field Id should be ID
velobike/profile.go:10:2: struct field UserId should be UserID
velobike/profile.go:16:2: struct field AvatarUrl should be AvatarURL
velobike/profile.go:22:2: struct field TariffId should be TariffID
velobike/velobike.go:12:2: const baseUrl should be baseURL
velobike/velobike.go:30:2: struct field SessionId should be SessionID
```

It's not possible to change it in the current major version (0.x.x) because it'll break back compatibility.
So, this issue should be fixed in the next major version (1.0.0).

--- a/README.md
+++ b/README.md
@@ -13,7 +13,7 @@ Client style based on the best practices of [google/go-github](https://github.co
 Semantic versioning: Marjor.Minor.Patch.
 Current version is 0.x.x.
 
-There are [few warnings from go linter](https://github.com/rumyantseva/go-velobike/issues/2) here. I'd like to fix these warnings, but it'll break backward compatibility. So, these changes are planned for the next major version of the library. Other issues might break back compatibility too, please check [1.0.0 milestone](https://github.com/rumyantseva/go-velobike/milestone/3) to find more information.
+There may be open issues that break backwards compatibility, please check [1.0.0 milestone](https://github.com/rumyantseva/go-velobike/milestone/3) to find more information.
 
 If you use this library, please vendor it using a package management tool to not to have backward compatibility problems in the future. 
 

--- a/README.md
+++ b/README.md
@@ -15,7 +15,7 @@ Current version is 0.x.x.
 
 There may be open issues that break backwards compatibility, please check [1.0.0 milestone](https://github.com/rumyantseva/go-velobike/milestone/3) to find more information.
 
-If you use this library, please vendor it using a package management tool to not to have backward compatibility problems in the future. 
+If you use this library, please vendor it using a package management tool to not to have backward compatibility problems in the future.
 
 If you have any questions, feel free to [create an issue](https://github.com/rumyantseva/go-velobike/issues/new), I'll try to answer as soon as possible.
 
@@ -32,9 +32,8 @@ Contributors are welcome! Feel free to improve this project or share your ideas 
 
 Contributors (unsorted):
 
-- Elena Grahovac: [@rumyantseva](https://github.com/rumyantseva)
-- Pavel Borzenkov: [@pborzenkov](https://github.com/pborzenkov)
-
+* Elena Grahovac: [@rumyantseva](https://github.com/rumyantseva)
+* Pavel Borzenkov: [@pborzenkov](https://github.com/pborzenkov)
 
 ## Roadmap
 

--- a/velobike/authorize.go
+++ b/velobike/authorize.go
@@ -8,7 +8,7 @@ type AuthorizeService struct {
 
 // Authorization describes the body of profile/authorize method response.
 type Authorization struct {
-	SessionId *string `json:"SessionId,omitempty"`
+	SessionID *string `json:"SessionId,omitempty"`
 }
 
 // Authorize method authorizes user and get receives token.

--- a/velobike/history.go
+++ b/velobike/history.go
@@ -16,7 +16,7 @@ type HistoryItem struct {
 	Type *string `json:"Type,omitempty"` // Possible types: "Ride", "Pay"
 
 	// Fields common for both "Ride" and "Pay" types
-	Id        *string  `json:"Id,omitempty"`
+	ID        *string  `json:"Id,omitempty"`
 	Price     *float64 `json:"Price,omitempty"`
 	Rejected  *bool    `json:"Rejected,omitempty"`
 	StartDate *string  `json:"StartDate,omitempty"` // layout: 2006-01-02T15:04:05

--- a/velobike/parkings.go
+++ b/velobike/parkings.go
@@ -14,7 +14,7 @@ type Parkings struct {
 type Parking struct {
 	Address     *string   `json:"Address,omitempty"`
 	FreePlaces  *int      `json:"FreePlaces,omitempty"`
-	Id          *string   `json:"Id,omitempty"`
+	ID          *string   `json:"Id,omitempty"`
 	IsFavourite *bool     `json:"IsFavourite,omitempty"`
 	IsLocked    *bool     `json:"IsLocked,omitempty"`
 	Position    *Position `json:"Position,omitempty"`

--- a/velobike/profile.go
+++ b/velobike/profile.go
@@ -7,19 +7,19 @@ type ProfileService struct {
 
 // Profile describes profile method response body.
 type Profile struct {
-	UserId                *string  `json:"UserId,omitempty"`
+	UserID                *string  `json:"UserId,omitempty"`
 	Email                 *string  `json:"Email,omitempty"`
 	PhoneNumber           *string  `json:"PhoneNumber,omitempty"`
 	RegisterDate          *string  `json:"RegisterDate,omitempty"`
 	FirstName             *string  `json:"FirstName,omitempty"`
 	LastName              *string  `json:"LastName,omitempty"`
-	AvatarUrl             *string  `json:"AvatarUrl,omitempty"`
+	AvatarURL             *string  `json:"AvatarUrl,omitempty"`
 	TroikaCardNumber      *string  `json:"TroikaCardNumber,omitempty"`
 	TroikaPrintCardNumber *string  `json:"TroikaPrintCardNumber,omitempty"`
 	Balance               *float64 `json:"Balance,omitempty"`
 	Holded                *bool    `json:"Holded,omitempty"`
 	HoldedAmount          *float64 `json:"HoldedAmount,omitempty"`
-	TariffId              *string  `json:"TariffId,omitempty"`
+	TariffID              *string  `json:"TariffId,omitempty"`
 	TariffStart           *string  `json:"TariffStart,omitempty"`
 	TariffEnd             *string  `json:"TariffEnd,omitempty"`
 }

--- a/velobike/velobike.go
+++ b/velobike/velobike.go
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	baseUrl = "http://apivelobike.velobike.ru"
+	baseURL = "http://apivelobike.velobike.ru"
 )
 
 // A Client manages communication with the API.
@@ -27,7 +27,7 @@ type Client struct {
 	History       *HistoryService
 
 	// Session ID for authorized user
-	SessionId *string
+	SessionID *string
 }
 
 // NewClient returns a new API client.
@@ -36,7 +36,7 @@ func NewClient(httpClient *http.Client) *Client {
 	if httpClient == nil {
 		httpClient = http.DefaultClient
 	}
-	baseURL, _ := url.Parse(baseUrl)
+	baseURL, _ := url.Parse(baseURL)
 
 	c := &Client{client: httpClient, BaseURL: baseURL}
 	c.Parkings = &ParkingsService{client: c}
@@ -74,8 +74,8 @@ func (c *Client) NewRequest(method, urlStr string, body interface{}) (*http.Requ
 		return nil, err
 	}
 
-	if c.SessionId != nil {
-		req.Header.Add("SessionId", *c.SessionId)
+	if c.SessionID != nil {
+		req.Header.Add("SessionID", *c.SessionID)
 	}
 
 	return req, nil

